lib/theme/colors.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+
+/// The names of the colors defined in [AllAppColors] are not as per the referenced source (Figma design)
+/// The corresponding color shade for each defined color has been documented.
+class AllAppColors {
+  /// Label/Light/300
+  static const Color label = Color(0xFF36383A);
+
+  /// Label/Light/200
+  static const Color hint = Color(0xFF8A8A8E);
+
+  /// Primary/Light/300
+  static const Color primary = Color(0xFF6A39F1);
+
+  /// Outline/Light/300
+  static const Color outline = Color(0xFFD1D1D6);
+
+  /// Outline/Light/200
+  static const Color divider = Color(0xFFE5E5EA);
+
+  /// Error/Light/300
+  static const Color error = Color(0xFFFF3B30);
+
+  /// Background/Light/100
+  static const Color disabled = Color(0xFFEEEEF3);
+}
```

Comment:
Since this class only has some constants and is `static`, this class should not have any default constructor.

What do I mean by that?
A class with some constant data should not have any named constructor. It should have a private constructor. We don't need multiple instances of the same anywhere around the app. 

You can define a constructor with 
`AppColors._()` 
So that you can prevent the caller from making any instance of this.

Reply:
There is one more advantage of this.
When you are writing this class name in the IDE and since the class has default constructor. IDE will suggest you to write a constructor first, you will end up having parentheses that you have to remove.
It's irritating if you do it again and again.

Reply:
This is great. When you don't define any constructors explicitly, the compiler defines the default unnamed constructor for you. But when you DO define any named constructor, the compiler doesn't do this favour of creating a default unnamed constructor anymore. But by declaring the named constructor private (with a leading `_` in its name), doesn't let the class to be instantiated outside of the class directly.